Modules/Order/Http/Controllers/OrderController.php

```diff
@@ -21,7 +21,10 @@
 class OrderController extends Controller
 {
 
-
+    /**
+     * @param Request $request
+     * @return mixed
+     */
```

Comment:
Move validation to Request class

[https://laravel.com/docs/9.x/validation#form-request-validation](https://laravel.com/docs/9.x/validation#form-request-validation)

Controller contains a lot of unused variables, remove them all.

Business logic should not be stored in controller class. In Http directory create "Services\OrderService.php" class and store logic there. Controller should only contain calls to service class methods.

Methods should not be too long, if logic is complicated break down to smaller functions